.github/workflows/adhoc.yml

```
@@ -10,6 +10,14 @@ on:
         description: "Asana task URL"
         required: false
         type: string
+      is-alpha:
+        description: "Is alpha build"
+        type: choice
+        required: true
+        default: 'true'
+        options:
+           - true
+           - false
```

GHA supports boolean type so we can use it here. It would then use a checkbox in the UI, instead of a dropdown:
```suggestion
      is-alpha:
        description: "Whether the build uses Alpha or Release configuration"
        type: boolean
        required: true
        default: true
```

---

I know, but I've seen people complaining that sometimes these behave like booleans, sometimes like strings, there's a whole thread https://github.com/actions/runner/issues/1483

---

Yes, that's why the call site still needs to treat them as strings :) just pointing it out because it's possible to use a checkbox here, although on a second thought we could make it a dropbox with more descriptive options, like "Configuration: Alpha/Release". I'm not strong on any of my suggestions though, as long as the message is clear.

---

I've tweaked the options and description a bit.

---

Thanks, that looks much clearer. Sorry to be a pain, but I'd also rename `is-alpha` now that it's not a boolean anymore. How about `build-type`? `configuration` is the most correct, but it also is a broad term in general and may be confusing.

---

Good suggestion, done :)